quipucords/api/reports/reports_gzip_renderer.py

```diff
@@ -53,6 +55,16 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
             deployments_csv_name: encode_content(deployments_csv, "csv"),
         }
 
+        # collect associated logs
+        scan_job_id = reports_dict.get("scan_job_id")
+        log_directory: Path = settings.LOG_DIRECTORY
+        log_files = list(log_directory.glob(f"scan-job-{scan_job_id}-*"))
+        if not log_files:
+            logger.warning(f"No logs were found for {report_id=}")
```

Comment:
Not necessarily something we need to change here, but normally all variables should be parameterized in logger calls so that their execution/concatenation is deferred or not run at all if the log level isn't appropriate.

Feel free to resolve this comment thread.

Reply:
ouch, I forgot about it. TY